api/src/pcapi/routes/pro/offers.py

```diff
@@ -65,13 +66,26 @@ def list_offers(query: offers_serialize.ListOffersQueryModel) -> offers_serializ
     api=blueprint.pro_private_schema,
 )
 def get_offer(offer_id: int) -> offers_serialize.GetIndividualOfferResponseModel:
-    load_all: offers_repository.OFFER_LOAD_OPTIONS = [
-        "mediations",
-        "product",
-        "price_category",
-        "venue",
-        "bookings_count",
-    ]
+
+    load_all: offers_repository.OFFER_LOAD_OPTIONS = (
+        [
+            "mediations",
+            "product",
+            "price_category",
+            "venue",
+            "bookings_count",
+            "offerer_address",
+        ]
+        if feature.FeatureToggle.WIP_ENABLE_OFFERER_ADDRESS.is_active()
+        else [
+            "mediations",
+            "product",
+            "price_category",
+            "venue",
+            "bookings_count",
+        ]
+    )
```

Comment:
[style] Plus simplement, et qui permet de diminuer le diff maintenant et quand on supprimer le feature flag : 

```suggestion
    load_all: offers_repository.OFFER_LOAD_OPTIONS = [
        "mediations",
        "product",
        "price_category",
        "venue",
        "bookings_count",
    ]
    if feature.FeatureToggle.WIP_ENABLE_OFFERER_ADDRESS.is_active():
        load_all.append("offerer_address")
```

Reply:
j'ai fait ça au début pylint n'aime pas. vu la définition du type.

Reply:
pylint ? Quelle est l'erreur ?

Reply:
mypy plutot:
 error: "Iterable[Literal['stock', 'mediations', 'product', 'price_category', 'venue', 'bookings_count', 'offerer_address']]" has no attribute "append" 
[attr-defined]

Reply:
Ce serait quand même plus lisible ainsi... On a le droit d'utiliser `type: ignore [attr-defined]`, exceptionnellement.

Reply:
ok. Comment on définit les exceptions? Ou s'est laissé à l'appréciation  ?